crates/re_renderer/src/wgpu_resources/static_resource_pool.rs

```diff
@@ -1,21 +1,34 @@
-use std::{collections::HashMap, hash::Hash, sync::atomic::Ordering};
+use std::{collections::HashMap, hash::Hash, ops::Deref};
 
+use parking_lot::{RwLock, RwLockReadGuard};
 use slotmap::{Key, SlotMap};
 
 use super::resource::{PoolError, ResourceStatistics};
 
-struct StoredResource<Res> {
+pub struct StoredResource<Res> {
     resource: Res,
     statistics: ResourceStatistics,
 }
 
-/// Generic resource pool for all resources that are fully described upon creation, i.e. never have any variable content.
+impl<Res> Deref for StoredResource<Res> {
+    type Target = Res;
+
+    fn deref(&self) -> &Self::Target {
```

Comment:
```suggestion
    #[inline]
    fn deref(&self) -> &Self::Target {
```

…we should add a lint for `#[inline]`ing trivial things to this. 

I actually added a lint for something similar in https://github.com/emilk/egui/pull/3557, but this should be added to clippy

Reply:
fixed in https://github.com/rerun-io/rerun/pull/4435